src/pfe/file-watcher/scripts/nodejs-container.sh

```diff
@@ -453,17 +466,29 @@ elif [ "$COMMAND" == "update" ]; then
 		fi
 	fi
 
-# Stop the application (not supported on Kubernetes)
+# Stop the application
 elif [ "$COMMAND" == "stop" ]; then
 	echo "Stopping node.js project $projectName"
-	$IMAGE_COMMAND exec $project /scripts/noderun.sh stop
+	if [[ "$IN_K8" == "true" ]]; then
+		POD_NAME="$( kubectl get po --selector=release=$project | grep 'Running' | cut -d ' ' -f 1 )"
+		kubectl cp /file-watcher/scripts/nodejsScripts/ "$POD_NAME":/scripts
+		kubectl exec "$POD_NAME" -- /scripts/noderun.sh stop
+	else
+		$IMAGE_COMMAND exec "$project" /scripts/noderun.sh stop
+	fi
 	$util updateAppState $PROJECT_ID $APP_STATE_STOPPING
-# Start the application (not supported on Kubernetes)
+# Start the application
 elif [ "$COMMAND" == "start" ]; then
 	echo "Starting node.js project $projectName"
 	# Clear the cache since restarting node will pick up any changes to package.json or nodemon.json
 	clearNodeCache
-	$IMAGE_COMMAND exec $project /scripts/noderun.sh start $AUTO_BUILD_ENABLED $START_MODE $HOST_OS
+	if [[ "$IN_K8" == "true" ]]; then
+		POD_NAME="$( kubectl get po --selector=release=$project | grep 'Running' | cut -d ' ' -f 1 )"
+		kubectl cp /file-watcher/scripts/nodejsScripts/ "$POD_NAME":/scripts
```

Comment:
if we're on kube, can we copy the scripts to the pod once; rather than doing it on start, stop, deployK8s..? Pls check the spring project on how we did this!

Reply:
I did that initially, following the spring project example, but I found that the container was sometimes unstable in Kube (for other reasons). So Kube would sometimes restart the container, without the scripts.

I decided to `cp` the scripts in everytime to ensured the scripts would work whether the container had restarted or not, and I did not notice any performance drop.

I think this way is more stable, unless you have strong objections?

Reply:
If you put the node scripts in the dockerfile and even if the pod node application pod restarts, the scripts are going to be there.. the node application is built using the dockerfile; so the pod will always have the scripts!

Reply:
ok. I have tried to get this working here https://github.com/rwalle61/codewind/pull/6 but am so far unsuccessful. Can you spot what needs to be done?

Is it that big a problem if we copy the files? I think this will take me some time to figure out because I am fairly new to Docker and Kube